### PR TITLE
installation instructions for the next person who uses this

### PR DESCRIPTION
**Pull request by aaaaargZombies**
A matching ruby version and gemfile.lock would have saved me about 40mins when installing this repo so lets fix that.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.6
```
Comment (ivan-kocienski-gfsc) [inline .ruby-version:1]: i would also add a `.ruby-gemset` with `mossleyheritage` so it doesn't use the global gemset for ruby 2.7.6

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Mossley Heritage Society's website
+
+This site is build with jekyll and can be viewed at https://mossleyheritage.org/
```
Comment (ivan-kocienski-gfsc) [inline README.md:3]: maybe a short requirements section covering ruby version, rbenv, ruby-gemset and ruby-build?

**Review by ivan-kocienski-gfsc — COMMENTED** [review]
Just a couple of points

File: README.md (hunk 2 above)
Comment (aaaaargZombies) [reply README.md:3]: would you be up for writing a suggestion for this, I think you know more than me about it.

Comment (aaaaargZombies) [reply README.md:3]: thankyou!

**Review by ivan-kocienski-gfsc — APPROVED** [review]
I am okay with this if you are, alfie

**Pull request merged**